setup.py

```diff
@@ -16,29 +16,33 @@
 else:
     executables = []
 
-
-_version_re = re.compile(r'__version__\s+=\s+(.*)')
-
-with open('alcli/__init__.py', 'rb') as f:
-    version = str(ast.literal_eval(_version_re.search(
-        f.read().decode('utf-8')).group(1)))
-
 with open('README.md') as readme_file:
     readme = readme_file.read()
 
 with open('HISTORY.rst') as history_file:
     history = history_file.read()
 
+try:
+    # This is to force definitions to be upgraded every build not related to the definitions change
+    sdk_pypi_url = "https://pypi.org/pypi/alertlogic-sdk-python/json"
+    with request.urlopen(sdk_pypi_url) as defs_rq:
+        defs_info = json.loads(defs_rq.read())
+    sdk_latest_version = defs_info['info']['version']
+    sdk_dependency = 'alertlogic-sdk-python>=' + sdk_latest_version
+except:
+    sdk_dependency = 'alertlogic-sdk-python>=1.0.26'
+
 requirements = [
-        'alertlogic-sdk-python==1.0.26',
+        sdk_dependency,
```

Comment:
I don't think this should be generated per install from some external source. Otherwise it becomes harder to attribute reported bugs to a given version of the cli / sdk. 

I'd go with automating this through some other means, like some bot that listens for new versions of the sdk and creates a PR against the CLI to bump the version. What do you think?

Reply:
This version gets into the wheel and embedded into the version.py according to the https://github.com/alertlogic/alcli/pull/21/files#diff-522adf759addbd3b193c74ca85243f7dR5

Therefore in the package it is fixed and enduser will see version like:
https://github.com/alertlogic/alcli/pull/21/files#diff-098099f1d35c6e7ed48387c9791aaf60R26
here i should add version of definitions as well yeah.